README.md

```diff
@@ -51,6 +51,3 @@ Give a ⭐️ to if you like this project!
 - Hat tip to anyone whose code was used
 - Inspiration
 - etc
```

Comment:
- [ ] Kindly gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).
- [ ] Also consider deploying the application using Heroku, Netlify, or GitHub Pages and add it in the readme.